### ISSUE TITLE
linting fails

### ISSUE THREAD

**Issue by codejamninja** [camptocamp/bivac#277]
I get the following error when running `make lint`.

```
can't load package: package github.com/golang/lint/golint: code in directory /usr/local/go/default/src/github.com/golang/lint/golint expects import "golang.org/x/lint/golint"
Makefile:14: recipe for target 'lint' failed
make: *** [lint] Error 1
```

**Comment by codejamninja** [comment]
I think the linting package has changed its import location.

**Comment by raphink** [comment]
Fixed in #278
